node_state_transitions/node_transition_lib.py

```diff
@@ -65,6 +65,18 @@
 
 from clearinghouse.website import settings
 
+# Beginning of part of fix for Issue #152
+# Perform django setup (required in Django 1.7+ (possibly 1.6.?+)) to make
+# db/models available. We need to do this if we are using a django version
+# that has django.setup() (1.7+?) This should happen before the repy patching
+# of builtins, which likely occurs in the add_dy_support call in the next line
+# of code. django.setup() expects to be able to use the builtin keyword
+# 'type', which is patched by repy, so we should do this here, before that
+# patching.
```

Comment:
In contrast to `polling/check_active_db_nodes.py`, this here `node_state_transitions/node_transition_lib.py` *does* import `repyportability` which indirectly overrides `type`. However, *again* no attempts are made to undo the override. (I thus wonder if it's really necessary to restore Python's `type` before running `django.setup()`).